lms/envs/common.py

```diff
@@ -2295,3 +2295,6 @@
 PROFILE_IMAGE_SECRET_KEY = 'placeholder secret key'
 PROFILE_IMAGE_MAX_BYTES = 1024 * 1024
 PROFILE_IMAGE_MIN_BYTES = 100
+
+# This is to check the domain in case of preview.
+PREVIEW_DOMAIN = 'preview'
```

Comment:
@symbolist 
Thank you for you informations.
Awesome. Thanks.
https://github.com/edx/edx-platform/pull/11661 is good for me.

And thanks for created a ticket but I can not see that link.
Deleted some reason? or being in private ticket?

Thanks,